### ISSUE TITLE
release date for generalized contrastive loss code?

### ISSUE THREAD

**Issue by DesaleF** [marialeyvallina/generalized_contrastive_loss#7]
Hello dear authors, Thank you for your interesting work. Are you going to release the code for GCL? I am trying to experiment on different contrastive learning and I want to try you proposed GCL but I could find the code. Can you please confirm me if the loss function code is going to be release or not in the near future?

**Comment by nicstrisc** [comment]
Hi,
we released the training script yesterday.
This is contextual with the acceptance of our paper at CVPR 2023 - see https://arxiv.org/abs/2303.11739v1
